src/stDDM.jl

```diff
@@ -0,0 +1,213 @@
+"""
+    stDDM{T<:Real} <: AbstractstDDM
+
+An object for the starting-time diffusion decision model.
+
+# Parameters 
+
+- `ν`:  vector of drift rates for attribute one and two
+- `α`:  evidence threshold
+- `τ`:  non-decision time 
+- `s`:  initial latency bias (positive for attribute two, negative for attribute one)
+- `z`:  initial evidence 
+- `η`:  vector of variability in drift rate for attribute one and two
+- `ρ`:  correlation between drift rate for attributes
+- `σ`:  diffusion noise
+- `Δt`: time step 
+
+# Constructors 
```

Comment:
In the constructors section, can you please conform to the following format:

```
# Constructors

Two constructors are defined below. The first constructor uses positional arguments, and is therefore order dependent:

    CDDM(ν, σ, η, α, τ)

The second constructor uses keywords with default values, and is not order dependent: 

    CDDM(; ν=[1,.5], η=[1,1], σ=1, α=1.5, τ=0.30) 
```

I added a verbal explanation based on Dominique's recommendation. He said some people who are not familiar with Julia may not understand the syntax for keyword arguments.